academy/ibc/ibc-token-lifecycle.md

```diff
@@ -0,0 +1,61 @@
+
+## Design constraints
+
+There are obvious usabililty requirements for a successful bridge:
+
+1. Easy to use
+2. Innexpensive
+3. Reasonably fast settlement time
+
+There are economic constraints that must not be violated:
+
+1. Does not alter the total supply or circulating supply of any token.
+
+## Solution components
+
+In summary, tokens that traverse the bridge are locked on the origin chain and then wrapped tokens are minted and emitted on the destination chain. A wrapped token on chain B is collateralized claim on an actual token on chain A. In reverse, wrapped tokens are redeemed and destroyed, which causes the bridge to send a signal to the other chain that causes an equal amount of collateral to be released. 
+
+<!-- TODO: insert flow diagram --->
+
+1. On-chain logic: Can be a Cosmos SDK Module or a smart contract (depending on which chain the component runs on). Each chain needs a component that can lock tokens in an escrow-like container and send a message to the bridge. In the reverse, it needs to receive instructions and release collateral. 
+2. The Oracle: An Oracle is a highly-available logical singleton. High-availability implies that the *implementation* usually consists of many redundant components. An Oracle must follow a strict protocol that strongly discourages misbehaviour. After all, an Oracle malfunction could dilute the supply of a token with serious, deleterious effects on projects exposed to this risk. 
+
+Together, the two sides of the on-chain logic and the Oracle form a bridge. Prior to the Inter-blockchain Protocol (IBC) it was not possible to generalize a bridge solution that would connect any chain to any other chain or allow assets to flow freely between them. While non-Cosmos chains do indeed require customized implementation in order to become compatible, Cosmos' bridge solution presents the architecture to make this possible. The Gravity bridge is one such implementation that connects Ethereum ERC20 tokens with the Cosmos blockchains. 
+
+## Cosmos' Oracle solution
+
+Cosmos introduces the concept of a Peg-Zone to reduce the uncertainty that arises from probabistic blockchains that rely on Proof-of-Work such as Ethereum. In summary, a Peg-Zone *deems* a transaction to be *final* after a certain number of confirmations.
+
+The Oracle service is implemented as a network of independent validators. These validators place funds at risk and are incentivized to report, and slash funds staked by other validators if those other validators are observed signing incorrect information. 
+
+Validator networks control the onchain components with strong multisignature protocols. That is to say, the on-chain components take action when an instruction is signed by a majority of the validators. This arrangement helps ensure that a maliceous validator cannot exploit the privileges that are entrusted to it. 
```

Comment:
"This arrangement helps ensure that a maliceous validator cannot exploit the privileges that are entrusted to it." --> why? please elaborate.